collectors/python.d.plugin/varnish/metadata.yaml

```diff
@@ -16,14 +16,22 @@ meta:
   most_popular: false
```

Comment:
can you add some keywords too? and maybe the link to varnish'es site, I can't edit that portion, GitHub won't allow me

Reply:
Oops will do